playwright.config.ts

```diff
@@ -0,0 +1,48 @@
+import { defineConfig, devices } from '@playwright/test'
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// import dotenv from 'dotenv'
+// dotenv.config({ path: path.resolve(__dirname, '.env') })
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  outputDir: './test_results/playwright/test-output',
+  testDir: './integration_tests/specs',
+  /* Maximum time one test can run for. (millis) */
+  timeout: 3 * 60 * 1000,
+  /* Maximum time test suite can run for. (millis) */
+  globalTimeout: 60 * 60 * 1000,
+  /* Run tests in files in parallel */
+  fullyParallel: false,
+  /* Ensure tests run consecutively due to inability to share wiremock instance */
+  workers: 1,
```

Comment:
needs to run strictly in parallel or wiremock gets confused

Reply:
I wonder if we can get a solution for this going, not running in parallel feels like it's skipping a chunk of the benefit we get from switching to Playwright